staging/src/k8s.io/kubectl/pkg/cmd/create/create.go

```diff
@@ -46,6 +46,133 @@ import (
 	"k8s.io/kubectl/pkg/util/templates"
 )
 
+// CreateFlags directly reflect the information that CLI is gathering via flags. They will be converted to Options,
+// which reflect the runtime requirements for the command.
+type CreateFlags struct {
+	PrintFlags  *genericclioptions.PrintFlags
+	RecordFlags *genericclioptions.RecordFlags
+	Recorder    genericclioptions.Recorder
+
+	fieldManager string
+
+	FilenameOptions  resource.FilenameOptions
+	Selector         string
+	EditBeforeCreate bool
+	Raw              string
+
+	genericclioptions.IOStreams
```

Comment:
Why CreateFlags have iostreams?

Reply:
I was following the example for `wait` command https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go#L95

and besides that I think it makes sense to have `IOStreams` in `CreateFlags` since flags would rely on `stdout` `stdin` to process